src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationController.java

```diff
@@ -51,15 +55,21 @@ public OcrValidationController(
     })
     public ResponseEntity<OcrValidationResponse> validateOcrData(
         @RequestHeader(name = "ServiceAuthorization", required = false) String serviceAuthHeader,
+        @PathVariable(name = "form-type", required = false) String formType,
         @Valid @RequestBody OcrDataValidationRequest request
     ) {
+        if (!Enums.getIfPresent(FormType.class, formType).isPresent()) {
```

Comment:
Dang, just checked and `Enums.getIfPreset` throws NPE if `formType` is null. It can be since not required.

Perhaps we could wrap it something like

```java
Optional.ofNullable(formType)
    .flatMap(type -> Enums.getIfPresent(FormType.class, type))
    .map(type -> ...)
    .orElseThrow(() -> new FormNotFoundException("Form type '" + formType + "' not found"))
```

how does it sound?